akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -64,6 +64,21 @@ akka.serialization.jackson {
     FAIL_ON_UNKNOWN_PROPERTIES = off
   }
 
+  # Configuration of the ObjectMapper mapper features.
+  # See com.fasterxml.jackson.databind.MapperFeature
+  # Enum values corresponding to the MapperFeature and their boolean value.
```

Comment:
> Enum values corresponding to the MapperFeature and their boolean value.

Seems a bit terse. I think we should also add examples to the comment.

Reply:
Thanks, @jrudolph. See if https://github.com/akka/akka/pull/27409/commits/1b666fef3e1c9ea004729d9ac7c75a676e793545 addresses this comment. :-)